x-pack/plugins/alerting/server/lib/process_alerts.ts

```diff
@@ -6,18 +6,22 @@
  */
 
 import { millisToNanos } from '@kbn/event-log-plugin/server';
-import { cloneDeep } from 'lodash';
+import { cloneDeep, drop } from 'lodash';
 import { Alert } from '../alert';
 import { AlertInstanceState, AlertInstanceContext } from '../types';
 
+const MAX_CAPACITY = 20;
```

Comment:
This value is duplicated in https://github.com/elastic/kibana/pull/144415/files#diff-0bfbaff15271561e51c55b7c2d1b218fd0c3e3323f43efd690906cdfd7b667eaR12
Should we have only one? What would happen if this capacity is not sync to the other one?

Reply:
Yes, I think you are right that it should be in one place. There is an issue to do this, https://github.com/elastic/kibana/issues/143529, where we will make these values configurable. I am hardcoding them for now until this work is complete. I can try to move them to one place in the meantime though!

Reply:
If it's tracked I'm fine with it as it is, thanks!